tools/export_program_info/export_program_info.vcxproj

```diff
@@ -214,6 +217,8 @@ $(OutputPath)export_program_info.exe</Command>
     </Link>
     <CustomBuildStep>
       <Command>cd /d $(OutputPath)
+copy ..\Release\usersim.dll .
```

Comment:
Since `export_program_info.exe` now links with `usersim.dll`, should we include `usersim.dll` in both nuget package and the MSI?

Reply:
usersim.dll is only for testing, so we should include it where test binaries are included.
Will check why export_program_info links with it...

Reply:
Answer: linkage is temporary in this PR until issue #2677 is addressed next milestone.   That is, ebpf_platform_user.cpp temporarily requires usersim.dll but won't once I do that cleanup.